recipe/meta.yaml

```diff
@@ -578,6 +579,9 @@ outputs:
         - {{ pin_subpackage('libopenvino-auto-plugin', exact=True) }}
         - {{ pin_subpackage('libopenvino-hetero-plugin', exact=True) }}
         - {{ pin_subpackage('libopenvino-auto-batch-plugin', exact=True) }}
+        - {{ pin_compatible('tbb-devel', max_pin='x.x') }}  # to property find libs via openvino.pc
```

Comment:
```suggestion
        - tbb-devel  # to property find libs via openvino.pc
```

Reply:
this is going to allow "any tbb-devel" unconstrained by any version. 

you should add the `{{ pin_compatible('tbb-devel', 'x' }}` to ensure that if 2021 is used at build time, then 2021 will be used at runtime.

Otherwise, maybe i'm missing something?

Reply:
TBB is BW compatible and they will not increase 2021 for a while.

But I've returned `{{ pin_compatible('tbb-devel', max_pin='x') }}`